### PR TITLE
Add initial support for Archlinux

### PR DESCRIPTION
**Pull request by TingPing**
(Sorry for the un-related commits they were just trivial problems)

### CODE REVIEW

**Comment by FrostyX** [comment]
Hi @TingPing,
thank you for the PR, that is awesome! :-)

Although I have few thoughts, that I would like to discuss. I will comment particular lines.

File: tracer/resources/applications.py [hunk 1]
```diff
@@ -116,7 +116,10 @@ def _append_application(default_attrs, specific_attrs={}):
 	@staticmethod
 	def _helper(app):
 		if app.type == Applications.TYPES["DAEMON"]:
-			return "service {0} restart".format(app.name)
+			if System.init_system() == "systemd":
```
Comment (FrostyX) [inline tracer/resources/applications.py]: I had it this way initially, but I was informed that `service` command is universal across the init systems and there is no need for using `service` there, `systemctl` there and `/etc/init.d/...` elsewhere. Do you have any issues with using `service` on Arch?

**Comment by FrostyX** [comment]
If you have AUR package or other way, how to properly install tracer on ArchLinux, could you please describe it in the docs? See <https://github.com/FrostyX/tracer/blob/master/doc/source/get-tracer.rst>. It would be awesome for other archers.

File: tracer/resources/applications.py (hunk 1 above)
Comment (TingPing) [inline tracer/resources/applications.py]: There is no `service` script provided by any package in Arch.

Comment (FrostyX) [inline tracer/resources/applications.py]: Aha, in this case feel free to use whatever you want on Arch :-). But not generally for systemd please, cause on Fedora we use systemd and `service` is prefered.

**Comment by FrostyX** [comment]
Thank you for the documentation.

File: tracer/resources/applications.py (hunk 1 above)
Comment (TingPing) [inline tracer/resources/applications.py]: No problem to make it Arch specific, but is that really true? I used Fedora for years and all documentation used `systemctl`. My understanding was that `service` existed enitrely for legacy reasons

Comment (FrostyX) [inline tracer/resources/applications.py]: Hmm, it seems that you are right. I am sorry, I probably had the wrong information.

Comment (TingPing) [inline tracer/resources/applications.py]: @FrostyX So was there anything else you wanted changed? Want me to revert that to all systemd platforms?

**Comment by FrostyX** [comment]
Thank you very much! :-)

I am sorry, that I didn't merged it, or replied earlier. I was on a trip and there was absolutely no time for internet. Just tried your version, it works on the Fedora correctly, so ... :+1:

**Pull request merged**